src/lib/sputter/net/port.cpp

```diff
@@ -0,0 +1,108 @@
+#include "port.h"
+
+#include <cstring>
+#include <iostream>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+using namespace sputter::net;
+
+UDPPort::UDPPort()
+    : socketHandle(-1) {
```

Comment:
I prefer to set default values in the header rather than in the constructor for brevity and having information in fewer places.

Also, stylistically this codebase is using the `m_` prefix for member variables.

So basically in port.h:
`int m_socketHandle = -1;`

And you can remove the default constructor declaration and definition.

Reply:
b51e447